packages/client/src/app/pages/work-page/section-page/section-page.component.html

```diff
@@ -38,7 +38,7 @@ <h1>{{ sectionData.title }}</h1>
                 </div>
             </div>
             <div class="authors-note" *ngIf="sectionData.authorsNote">
-                <h3>A word from the author...</h3>
+                <h2 class="authors-note-header">A word from the author...</h2>
```

Comment:
The `h3` -> `h2` change for the author's note box header is purely semantic--it has the same visual appearance, but I think it makes more sense to consider it an `h2` instead of skipping from the section title's `h1` to an `h3`. It's nicer for screen readers, too. according to MDN.